src/Intervals.jl

```diff
@@ -28,6 +28,10 @@ export AbstractInterval,
        isclosed,
        anchor,
        span,
-       ..
+       disjoint_less_than,
+       disjoint_greater_than,
```

Comment:
If we expand the amount of greater/less than functions I could possibly see us going with `less_than_disjoint` for easy discoverability. I'm fine with leaving this and just doing deprecations if we decide to switch things.

Reply:
Good point. I'll make this change now.